src/doc/trpl/dining-philosophers.md

```diff
@@ -310,13 +310,13 @@ use std::thread;
 from the standard library, and so we need to `use` it.
 
 ```rust,ignore
-    fn eat(&self) {
-        println!("{} is eating.", self.name);
+fn eat(&self) {
+    println!("{} is eating.", self.name);
 
-        thread::sleep_ms(1000);
+    thread::sleep_ms(1000);
 
-        println!("{} is done eating.", self.name);
-    }
+    println!("{} is done eating.", self.name);
+}
 ```
```

Comment:
I see indentation of some previous example on this wasn't kept. Should I preserve it as well?